inc/components/class-head.php

```diff
@@ -19,6 +19,22 @@ class Head extends Component {
 	 */
 	public $name = 'head';
 
+	/**
+	 * Initalize this component.
+	 *
+	 * @param null|\WP_Query $wp_query WP_Query object or null.
+	 */
+	public function __construct( $wp_query = null ) {
+		parent::__construct();
+
+		// Set the default title.
+		$this->set_title( get_bloginfo( 'name' ) );
+
+		if ( $wp_query instanceof \WP_Query ) {
```

Comment:
Every other component has kept the parent constructor interface, and then the query can be applied with a setter. Are you sure you want to break that consistency?